src/dynarec/arm64/dynarec_arm64_66f30f.c

```diff
@@ -55,6 +55,58 @@ uintptr_t dynarec64_66F30F(dynarec_arm_t* dyn, uintptr_t addr, uintptr_t ip, int
     #endif
 
     switch(opcode) {
+        case 0xB8:
+            INST_NAME("POPCNT Gw, Ew");
+            SETFLAGS(X_ALL, SF_SET);
+            SET_DFNONE();
+            nextop = F8;
+            GETGW(x2);
+            GETEW(x1, 0);
+            v1 = fpu_get_scratch(dyn, ninst);
+            VMOVQDfrom(v1, 0, ed);
```

Comment:
you might need to VEOR v1 first, as this just replace 1 element on the vector, leving the other untouched.